openmetadata-ui/src/main/resources/ui/src/utils/DatabaseSchemaDetailsUtils.tsx

```diff
@@ -203,3 +204,23 @@ export const getDatabaseSchemaWidgetsFromKey = (widgetConfig: WidgetConfig) => {
     />
   );
 };
+
+export function buildSchemaQueryFilter(
+  field: string,
+  fieldValue: string,
+  searchValue?: string
+) {
+  return getTermQuery(
```

Comment:
can't we directly use this `getTermQuery` method instead of creating a new method ?